src/expedited.cpp

```diff
@@ -286,3 +290,14 @@ void SendExpeditedBlock(const CBlock &block, const CNode *skip)
     }
     // else, nothing to do
 }
+
+bool IsExpeditedNode(const CNode *pfrom)
+{
+    // Check if this node is an expedited node
+    BOOST_FOREACH (const std::string &strAddrNode, mapMultiArgs["-expeditedblock"])
+    {
+        if (pfrom->addrName == strAddrNode)
+            return true;
+    }
+    return false;
+}
```

Comment:
expedited partners can be set via RPC call and we'll want an automatic option in the future.  So to discover whether a node is expedited, please use one of these global vars.
std::vector<CNode*> xpeditedBlk; // (256,(CNode*)NULL);    // Who requested expedited blocks from us
std::vector<CNode*> xpeditedBlkUp; //(256,(CNode*)NULL);  // Who we requested expedited blocks from
std::vector<CNode*> xpeditedTxn; // (256,(CNode*)NULL);